edge-modules/ModuleLib/networkcontrollerreuslt/NetworkControllerResult.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace ModuleUtil.NetworkControllerResult
+{
+    using Newtonsoft.Json;
+    public class NetworkControllerResult
+    {
+        public string TrackingId { get; set; }
+
+        public string Operation { get; set; }
+
+        public string OperationStatus { get; set; }
+
+        public NetworkStatus NetworkStatus { get; set; }
+
+        public bool Enabled { get; set; }
```

Comment:
It looks like the naming is a bit confusing, I would say NetworkStatus should be more like NetworkControllerType and Enabled is should be NetworkStatus enum with Enabled/Disabled.